src/Commands/SiteCloneCommand.php

```diff
@@ -522,7 +517,12 @@ private function wpcliSearchReplace()
             ]
         );
 
-        $this->destination['env_raw']->sendCommandViaSsh('wp search-replace ' . $this->source['pantheon_domain'] . ' ' . $this->destination['pantheon_domain']);
+        $WPCommand = new WPCommand();
+
+        $WPCommand->wpCommand(
```

Comment:
I couldn't get [`wpCommand`](https://github.com/pantheon-systems/terminus/blob/master/src/Commands/Remote/WPCommand.php#L34) to work - it kept failing with the following error:

```sh
Using wp-cli search-replace to replace dev-wp-quicksilver-demo.pantheonsite.io with clone-test-december-couch-coding.pantheonsite.io on b28f9ea5-a2d7-445b-b047-2c174a276205.clone-test as WordPress stores URLs in the database. You may need to run the search-replace manually if you are using custom domains.

PHP Fatal error:  Uncaught Error: Call to a member function get() on null in /Users/andrewtaylor/pantheon-cli/src/Site/SiteAwareTrait.php:46
Stack trace:
#0 /Users/andrewtaylor/pantheon-cli/src/Site/SiteAwareTrait.php(87): Pantheon\Terminus\Commands\Remote\SSHBaseCommand->getSite('december-couch-...')
#1 /Users/andrewtaylor/pantheon-cli/src/Commands/Remote/SSHBaseCommand.php(49): Pantheon\Terminus\Commands\Remote\SSHBaseCommand->getSiteEnv('december-couch-...')
#2 /Users/andrewtaylor/pantheon-cli/src/Commands/Remote/WPCommand.php(36): Pantheon\Terminus\Commands\Remote\SSHBaseCommand->prepareEnvironment('december-couch-...')
#3 /Users/andrewtaylor/.terminus/plugins/terminus-site-clone-plugin/src/Commands/SiteCloneCommand.php(523): Pantheon\Terminus\Commands\Remote\WPCommand->wpCommand('december-couch-...', Array, Array)
#4 /Users/andrewtaylor/.terminus/plugins/terminus-site-clone-plugin/src/Commands/SiteCloneCommand.php(387): Pantheon\TerminusSiteClone\Commands\SiteCloneCommand->wpcliSearchReplace()
#5 /Users/andrewtaylor/.term in /Users/andrewtaylor/pantheon-cli/src/Site/SiteAwareTrait.php on line 46
```